cms/djangoapps/contentstore/views/tests/test_tabs.py

```diff
@@ -71,6 +75,18 @@ def test_view_index(self):
         resp = self.client.get_html(self.url)
         self.assertContains(resp, 'course-nav-list')
 
+    @ddt.data(
+        # toggle active, assert count
+        (True, 1),
+        (False, 0),
+    )
+    @ddt.unpack
+    def test_pages_and_resources_toggle(self, toggle_active, assert_count):
+        """Basic check that the Pages page responds correctly"""
```

Comment:
```suggestion
        """Basic check that the Pages and Resources page responds correctly"""
```

Reply:
This is actually about the pages page in the current studio, not the pages and resources page, so this is correct.